### DISCUSSION TITLE
Is there a way to customize the pages header and breadcrumbs ?

### DISCUSSION THREAD

**Discussion by hazwan91** [filamentphp/filament#8404] [Help] [answered]
![Screenshot 2023-09-11 152146](https://github.com/filamentphp/filament/assets/97268905/0fed6c5d-efc2-4b57-9805-7eefcf365da9)

Is there a way to change this part?

I can't see it on the docs and not sure if its possible.

**Comment by danharrin** [comment, marked as answer]
Override `getHeading()` or `getBreadcrumbs()` on the page class

**Reply by quasiperfect** [reply]
hi @danharrin do we have something similar to getHeading() for resources ?

**Comment by hazwan91** [comment]
For breadcrumb it was ok, i can refer on `Filament\Resources\Resource`. It use getBreadcrumb without `s`.
```
public static function getBreadcrumb(): string
    {
        return 'Custom Page Header';
    }
```

However, i can't find the getHeading function and try to add it in but no change occured.

public static function getHeading(): string
    {
        return 'Test Custom Header';
    }

or 

`protected static ?string $heading = 'Custom Page Heading';`

No changes at all..

**Comment by hazwan91** [comment]
Nevermind. I put in wrong file...works now.. Thanks Dan

**Comment by kurniawan750** [comment]
hello,
it happen to me today getBreadcrumb is work for me, but :
`  public static function getHeading(): string
    {
        return 'Test Custom Header';
    }
`
or 
`protected static ?string $heading = 'Custom Page Heading';`
does not seem change my table header ?

**Comment by hazwan91** [comment]
I just put it like this on `List Pages` **AND NOT** at the `Resource Page`.

```
public function getBreadcrumbs(): array
    {
        return [
            'districts' => 'Daerah',
            'Senarai'
        ];
    }

    public function getHeading(): string
    {
        return 'Daerah';
    }
```

**Reply by kurniawan750** [reply]
thank again for reply i can managed to change header with :
`    public static ?string $pluralModelLabel = 'Database 2022';
`
